en/access-dashboard.md

```diff
@@ -156,6 +156,35 @@ type: kubernetes.io/tls
 
 After Ingress is deployed, visit <https://{host}/dashboard> to access TiDB Dashboard.
 
+### Use NodePort Service
+
+Because you must access `ingress` via the domain name, it might be difficult to use `ingress` in some scenarios. In this case, to access and use TiDB Dashboard, you can add the `Service` of `NodePort` type.
```

Comment:
```suggestion
Because `ingress` must be accessed via the domain name, it might be difficult to use `ingress` in some scenarios. In this case, to access and use TiDB Dashboard, you can add the `Service` of `NodePort` type.
```